app/core/ViewLoadTimer.coffee

```diff
@@ -24,6 +24,7 @@ class ViewLoadTimer
       while views.length
         subView = views.pop()
         views = views.concat(_.values(subView.subviews))
+        # TODO: Include Vue data loading in page load times
```

Comment:
Mostly that data loading hasn't been standardized into one particular way of doing it yet (that can be reached into from the outside to check).